src/main/java/io/pravega/connectors/flink/util/FlinkPravegaParams.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.connectors.flink.util;
+
+import com.google.common.collect.Sets;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.connectors.flink.FlinkPravegaReader;
+import io.pravega.connectors.flink.FlinkPravegaWriter;
+import io.pravega.connectors.flink.PravegaEventRouter;
+import io.pravega.connectors.flink.serialization.PravegaSerialization;
+import java.io.Serializable;
+import java.net.URI;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.streaming.util.serialization.DeserializationSchema;
+import org.apache.flink.streaming.util.serialization.SerializationSchema;
+
+/**
+ * Convience class for working with pravega parameters coming in as parameters.
```

Comment:
Typo in `Convience`. 

I think `pravega parameters coming in as parameters` could be improved.